fdbserver/DeltaTree.h

```diff
@@ -1709,7 +1713,13 @@ struct DeltaTree2 {
 		} else {
 			nodeBytesUsed = 0;
 		}
+
+		ASSERT(size() <= spaceAvailable);
 		nodeBytesFree = spaceAvailable - size();
+
+		// Mark unused available space as defined
+		VALGRIND_MAKE_MEM_DEFINED((uint8_t*)this + size(), nodeBytesFree);
```

Comment:
The extra bytes are not zeroed out, this is just to tell valgrind to not complain if the bytes are read as part of the page checksum.  I suppose we should zero out the unused bytes though to prevent writing random memory to disk.